doc/releases/pacific.rst

```diff
@@ -2,6 +2,529 @@
 Pacific
 =======
 
+v16.2.11 Pacific
+================
+
+This is the eleventh backport release in the Pacific series.
+
+Notable Changes
+---------------
```

Comment:
nit: it will be nice to add a blank line here and between the bulleted items

Reply:
@yuriw this still needs to be fixed ^